src/webgpu/api/operation/vertex_state.spec.ts

```diff
@@ -171,8 +184,19 @@ class IndexFormatTest extends GPUTest {
       1.0, 1.0, 0.0, 1.0,
       -1.0, -1.0, 0.0, 1.0]
     );
+
+    let bufferSize = vertexArray.byteLength;
+    // Create a large enough buffer for uint16 primitive restart value used as a big index.
+    if (
+      indexFormat === 'uint16' &&
+      primitiveTopology !== 'line-strip' &&
+      primitiveTopology !== 'triangle-strip'
+    ) {
+      bufferSize = 0xffff * 16;
```

Comment:
This is 0xffff vertex elements, but you need one more.

Reply:
Why? I'm not sure I followed.

Reply:
Index 0xffff is the last element in an array of size 0x10000.

Obsolete with the suggestion about doing this without a vertex buffer, though.